fixtures/suspense/sku.config.js

```diff
@@ -0,0 +1,16 @@
+import { makeStableHashes } from '@sku-private/test-utils';
+
+export default {
+  sites: [
+    {
+      name: 'au',
+      host: 'dev.seek.com.au',
+      routes: [{ route: '/', name: 'home' }],
+    },
+  ],
+  environments: ['production'],
+  port: 8202,
```

Comment:
This clashes with the `multiple-routes` fixture's port. It should be different otherwise it can't be tested in parallel with other fixtures.